app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+
+  def self.update_comment_counter(post_id)
+    post = Post.find(post_id)
+    post.update(comment_counter: post.comments.count)
+  end
```

Comment:
- [ ] [OPTIONAL] Please consider using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comment_counter` method every time a new comment is saved to database, then make the update_comment_counter method private